frontend/test/metabase/scenarios/pulse.cy.spec.js

```diff
@@ -0,0 +1,66 @@
+import { signInAsAdmin, restore } from "__support__/cypress";
+
+const MOCK_PULSE_FORM_INPUT = {
+  channels: {
+    email: {
+      type: "email",
+      name: "Email",
+      allows_recipients: true,
+      configured: true,
+      recipients: ["user", "email"],
+      schedules: ["daily", "weekly", "monthly"],
+    },
+  },
+};
+
+describe("pulse", () => {
+  before(restore);
+  beforeEach(signInAsAdmin);
+  it("should be able get to the new pulse page from the nav bar", () => {
+    cy.visit("/");
+
+    cy.get(".Icon-add").click();
+    cy.contains("New pulse").click();
+
+    cy.url().should("match", /\/pulse\/create$/);
+  });
+
+  it("should create a new pulse", () => {
+    cy.server();
+    cy.route("GET", "/api/pulse/form_input", MOCK_PULSE_FORM_INPUT);
```

Comment:
Why mock this? Would email need to be set up otherwise?

Reply:
Correct.